Cod3rsGrowth.forms/FormEditarAnime.cs

```diff
@@ -0,0 +1,136 @@
+        {
+            var listaGeneros = _generoServico.ObterTodos();
+            foreach (var genero in listaGeneros)
+            {
+                clGeneros.Items.Add(genero.Nome);
+            }
+            clGeneros.MultiColumn = true;
+            List<AnimeGenero> listaAnimeGeneros = _animeGeneroServico.ObterTodos(_animeModificado.Id);
+
+            for (int i = POSICAO_INICIAL_NA_LISTA; i < listaAnimeGeneros.Count; i++)
+            {
+                var genero = _generoServico.ObterPorId(listaAnimeGeneros[i].IdGenero);
+                for (int j = POSICAO_INICIAL_NA_LISTA; j < clGeneros.Items.Count; j++)
+                {                 
+                    if (genero.Nome.Equals(clGeneros.Items[j]))
+                    {
+                        clGeneros.SetItemChecked(j, true);
+                    }
+                }
+            }
+        }
+        private void PreencherComboBoxStatus()
+        {
+            cbStatusDeExibicao.DataSource = new List<string>() { "EmExibição", "Previsto", "Concluído" };
+        }
+        private void AoClicarEmSalvar(object sender, EventArgs e)
+        {
+            try
+            {
+                AoClicarEmAtualizarAnime();
+                AoClicarEmEditarAnimeGenero();
+                DialogResult = DialogResult.OK;
+                this.Close();
+            }
+            catch (ValidationException ex)
+            {
+                string result = "";
+                foreach (var erro in ex.Errors)
+                {
+                    result += erro.ErrorMessage + "\n";
+                }
+                MessageBox.Show(result);
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.Message);
+            }
+        }
+
+        private void AoClicarEmCancelar(object sender, EventArgs e)
+        {
+            this.Close();
+        }
+        private void AoClicarEmAtualizarAnime()
+        {
+            Anime.Status status = new Anime.Status();
+            if (cbStatusDeExibicao.SelectedIndex == INDEX_EM_EXIBICAO)
+            {
+                status = Anime.Status.EmExibicao;
+            }
+            else if (cbStatusDeExibicao.SelectedIndex == INDEX_PREVISTO)
+                status = Anime.Status.Previsto;
+            else if (cbStatusDeExibicao.SelectedIndex == INDEX_CONCLUIDO)
+                status = Anime.Status.Concluido;
+            _animeServico.Atualizar(new Anime()
+            {
+                Id = Convert.ToInt32(textID.Text),
+                Nome = textNome.Text,
+                Sinopse = textSinopse.Text,
+                Nota = Convert.ToDecimal(campoNota.Value),
+                DataLancamento = dtpDataDeLancamento.Value,
+                StatusDeExibicao = status
+            });
+        }
+        private void AoClicarEmEditarAnimeGenero()
+        {          
+            int idAnime = _animeModificado.Id;
+            _animeGeneroServico.Deletar(idAnime);
```

Comment:
vc nao pode deletar assim nesse caso vc vai manter os que ja tem cadastrar os novos e deletar os que for removido